test/test_nestedtensor.py

```diff
@@ -1936,6 +1948,30 @@ def test_empty_like(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, "empty_like only supports contiguous memory format for Nested Tensors"):
             nt_empty = torch.empty_like(nt_noncont)
 
+    def test_fake_nt_creation(self, device):
+        with CrossRefNestedFakeMode():
```

Comment:
Are you going to take a page from test_sparse.py and run ALL the nested tensor tests under cross ref in the cross ref config?

Reply:
I like this idea; there will be lots of exFails due to missing meta support but I think it's okay